src/applications/personalization/profile/tests/e2e/personal-health-care-contacts/contacts.cypress.spec.js

```diff
@@ -25,6 +25,22 @@ describe('Personal health care contacts -- feature enabled', () => {
     cy.intercept('GET', '/v0/feature_toggles*', featureToggles);
   });
 
+  it('links from the hub page', () => {
+    cy.intercept('GET', '/v0/profile/contacts', contacts);
+    cy.login(loa3User72);
+    cy.visit(PROFILE_PATHS.PROFILE_ROOT);
+    cy.get('a[href$="/profile/contacts"]').should('exist');
```

Comment:
Consider using findByRole and look for the text of the link instead. This will be more inline with the preference of testing what the user sees.